dangerfile.ts

```diff
@@ -0,0 +1,8 @@
+import { danger, warn } from 'danger'
+
+// Check for a CHANGELOG entry
+const hasChangelog = danger.git.modified_files.some(f => f === 'CHANGELOG.md')
```

Comment:
note, `FocusZone` has changelog file as well - probably, it is still better to improve this filter before merging, for the sake of making the logic of warning to be consistent

![image](https://user-images.githubusercontent.com/9024564/48628651-c30de980-e9c8-11e8-9d36-a5155a1f238c.png)

Reply:
Ahh, good catch, I haven't realised that.

Reply:
Actually, `modified_files` contains paths from repo root so the `hasChangelog` is true only if `CHANGELOG.md` in root is changed.